TodoApp/TodoApp.Api/Controllers/ItemsController.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using System.Web.Http;
+using System.Web.Http.Description;
+using Microsoft.Web.Http;
+using TodoApp.Api.Models;
+
+namespace TodoApp.Api.Controllers
+{
+    [ApiVersion("1.0")]
+    [RoutePrefix("api/v{version:apiVersion}/items")]
+    public class ItemsController : ApiController
+    {
+        internal static readonly IList<Item> Items = new List<Item>
```

Comment:
You can change this collection to an array and return a specific item from an array in the methods.